query/src/test/java/org/infinispan/query/dsl/embedded/testdomain/hsearch/UserHS.java

```diff
@@ -35,7 +35,7 @@
    @Field(store = Store.YES, analyze = Analyze.NO, indexNullAs = Field.DEFAULT_NULL_TOKEN)
    private String surname;
 
-   @Field(store = Store.YES, analyze = Analyze.NO, indexNullAs = Field.DEFAULT_NULL_TOKEN)
```

Comment:
this change is needed in the integrationstests also, to solve one of the CI failures.